src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -108,7 +108,6 @@ public async Task RegisterIPEndPoint_DynamicPort_Success(IPEndPoint endPoint, st
         [ConditionalTheory]
         [MemberData(nameof(IPEndPointRegistrationDataPort443))]
         [IPv6SupportedCondition]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/2711")]
```

Comment:
Check the updated guidelines. Do not unquarantine tests that have open issues.

Reply:
Thanks, I hadn't seen that. Revert'ed.